app/controllers/events_controller.rb

```diff
@@ -4,10 +4,10 @@ class EventsController < ApplicationController
   before_action :is_logged_in?, only: [:student, :coach]
 
   def index
-    events = [ Workshop.past.all ]
-    events << Course.past.all
-    events << Meeting.past.all
-    events << Event.past.all
+    events = [ Workshop.recent.all ]
+    events << Course.recent.all
```

Comment:
@dansteele we do not actually use courses so I think this could be removed for now. The types we use are Workshops, Meetings and Events.